### PR TITLE
6550 Create Base

### PR DESCRIPTION
**Pull request by Hiepsisoma**
Create Base class MVP.
Create Model Genre, Track.
https://edu-redmine.framgia.vn/issues/6550

### CODE REVIEW

**Comment by Hiepsisoma** [comment]
ready

File: app/src/main/java/vn/framgia/phamthehung/soundcloud/data/model/Genre.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package vn.framgia.phamthehung.soundcloud.data.model;
+
+import java.io.Serializable;
+
+public class Genre implements Serializable {
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/vn/framgia/phamthehung/soundcloud/data/model/Genre.java]: a suggest dùng Parcelable

File: app/src/main/java/vn/framgia/phamthehung/soundcloud/data/model/Track.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
+package vn.framgia.phamthehung.soundcloud.data.model;
+
+import java.io.Serializable;
+
+public class Track implements Serializable {
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/vn/framgia/phamthehung/soundcloud/data/model/Track.java]: tương tự

**Comment by Hiepsisoma** [comment]
ready

**Comment by Hiepsisoma** [comment]
ready

**Review by DoanVanToan — APPROVED** [review]

**Pull request merged**